doc/api-reference.rst

```diff
@@ -4,8 +4,26 @@ API Reference
 Portal interfaces are available to sandboxed applications with the default
 filtered session bus access of Flatpak.
 
-Unless otherwise specified, they appear under the bus name ``org.freedesktop.portal.Desktop``
-and the object path ``/org/freedesktop/portal/desktop`` on the session bus.
+Desktop portals appear under the bus name ``org.freedesktop.portal.Desktop``
+and the object path ``/org/freedesktop/portal/desktop`` on the session bus,
+unless specified otherwise.
+
+Apps running on the host system have access to a special interface for
+registering themselves with XDG Desktop Portal. Registering a host app with
+XDG Desktop Portal overwrites the automatic detection based on the
+`XDG cgroup pathname standardization for applications
+<https://systemd.io/DESKTOP_ENVIRONMENTS/#xdg-standardization-for-applications>`_.
+This might improve the user experience when the host app was launched in a way
+that doesn't follow the standard. See
+:doc:`org.freedesktop.host.portal.Registry <doc-org.freedesktop.host.portal.Registry>`
+
+Disclaimer: The host app calls to the registry will eventually be replacable
+with the app placing itself in a cgroup named according to specific naming
+conventions. It will be documented in
+:doc:`org.freedesktop.host.portal.Registry <doc-org.freedesktop.host.portal.Registry>`
+when this will be possible.
```

Comment:
```suggestion
Disclaimer: The host app registry will eventually be deprecated and removed. App launchers, or apps themselves, should place the app in a cgroup named according to specific naming conventions. When the host app registry becomes deprecated, the details of the replacement will be documented in :doc:`org.freedesktop.host.portal.Registry <doc-org.freedesktop.host.portal.Registry>`.
```

Reply:
Even though we already know the conventions that shall be used right now. We should only deprecated the registry once we have managed to get a majority of applications to use the new convention (one way or another).

Reply:
Made the deprecation note a bit softer, i.e. "deprecated and may be removed" to be more about intention and not a 100% prediction of the future - the important part is that apps eventually should be able to remove the call and rely on cgroups.